lib/puppetserver/ca/config/puppet.rb

```diff
@@ -239,6 +245,26 @@ def parse_crl_usage(setting)
             :ignore
           end
         end
+
+        def update_for_server_list!(settings)
```

Comment:
So if I'm reading this right, we're giving precedence to `server` here. In Puppet, `server_list` takes precedence over `server`. See https://github.com/puppetlabs/puppet/blob/master/lib/puppet/defaults.rb#L1456-L1459 and https://github.com/puppetlabs/puppet/blob/master/lib/puppet/configurer.rb#L217-L238. I think we want to preserve that behavior here.

Reply:
Oh, good catch.